bin/ember-watson.js

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env node
+
+/* jshint multistr: true */
+process.chdir('/code');
+
+var exec = require('child_process').execSync;
+var fs = require('fs');
+var Watson = require('ember-watson');
+var CODE_DIR = 'app';
+try {
+  fs.accessSync('app');
+  CODE_DIR ='app';
+} catch (e) {
+  CODE_DIR ='addon';
+}
+
+var formulae = [{
+  name: 'QUnit 1.x tests detected',
+  command: 'transformQUnitTest',
+  path: 'tests',
```

Comment:
It looks like most of these commands supply their own default path: https://github.com/abuiles/ember-watson/blob/master/lib/commands/upgrade-qunit-tests.js#L17

Would it make sense to rely on the default behavior instead of hard-coding path overrides?

Would `watson[formula.command](null, true)` work?